src/apps/components/run.js

```diff
@@ -83,21 +83,12 @@ class RunApp extends React.Component {
     render() {
         return (
             <div className="app-main">
-                <div className="main-view">
-                    <JuttleViewer bundle={this.props.bundle} />
+                <RunHeader {...this.props} onRunClick={this.handleRunClick} ref="runHeader" />
+                <div className="run-main">
                     <div ref="juttleSource"></div>
                     <div ref="juttleViewLayout"></div>
                     <ErrorView error={this.props.error} />
                 </div>
-                <div className="right-rail">
-                    <div ref="juttleInputsContainer" onKeyDown={this._onInputContainerKeyDown}></div>
```

Comment:
I think `this._onInputContainerKeyDown` can be cleaned up from this component, but make sure it works in the header component.